src/chrome/content/rules/ebay.com.xml

```diff
@@ -0,0 +1,86 @@
+<!--
+
+	Other eBay rulesets:
+
+		- ebay.at
+		- ebay.be
+		- ebay.ca
+		- ebay.ch
+		- ebay.co.uk
+		- ebay.com
```

Comment:
Please remove the reference to itself and add `.xml` to all of them.
Not all ccTLD rulesets do exist. Do you plan to add them all before we merge this?

The previous references need to be updated too:
```
- CSdata1.com.xml
- ebayclassifieds.com.xml
- Gittigidiyor.xml
- gumtree.com.xml
- Shopping.com.xml
- Shoppingshadow.com.xml
- mobile.de.xml
```

Reply:
No, currently only those ccTLD. I'll just add `ebayclassifiedsgroup.com.xml`.
https://github.com/EFForg/https-everywhere/issues/13603#issuecomment-348856123